crates/bitwarden-core/src/key_management/crypto_client.rs

```diff
@@ -80,6 +80,16 @@ impl CryptoClient {
     ) -> Result<UserCryptoV2KeysResponse, StatefulCryptoError> {
         get_v2_rotated_account_keys(&self.client)
     }
+
+    /// Update the user's kdf settings, which will re-encrypt the user's encryption key with the new
+    /// kdf settings. This returns the new encrypted user key and the new password hash.
+    pub fn update_kdf(
+        &self,
+        password: String,
+        kdf: Kdf,
+    ) -> Result<UpdateKdfResponse, CryptoClientError> {
+        update_kdf(&self.client, &password, &kdf)
+    }
```

Comment:
The name is a bit confusing, since it does not update (state), but make's the update kdf (response).
```suggestion
    /// Update the user's kdf settings, which will re-encrypt the user's encryption key with the new
    /// kdf settings. This returns the new encrypted user key and the new password hash.
    pub fn make_update_kdf(
        &self,
        password: String,
        kdf: Kdf,
    ) -> Result<UpdateKdfResponse, CryptoClientError> {
        make_update_kdf(&self.client, &password, &kdf)
    }
```

Reply:
Hmm, I think your suggestion definitely is the better naming. However, this would break convention with what's existing in the file where we use update_password which also does not actually update state. In my mental list of priorities, consistency within a file ranks higher than (without context) better naming. 

So, how do we want to proceed:
- Go with make_update_kdf , but at the same time add make_update_password that just routes through to update_password, with a note / ticket for mobile to update to the new function, after which we will remove update_password
- Just go with update_kdf and accept the naming is somewhat confusing, but gain concistency
- Anything else?

Reply:
I would vote for `make_update_kdf` and `make_update_password`